docs/resources/aws_ssm_document.md

```diff
@@ -0,0 +1,89 @@
+---
+title: About the aws_ssm_document Resource
+platform: aws
+---
+
+# aws\_ssm\_document
+
+Use the `aws_ssm_document` InSpec audit resource to test properties of a ssm document.
+
+## Syntax
+
+ An `aws_ssm_document` resource block uses the parameter to select a ssm document.
+
+    describe aws_ssm_document(name: 'document-name-1234') do
+      it { should exist }
+    end
+
+
+#### Parameters
+
+##### name _(required)_
+
+This resource accepts a single parameter, the SSM Document Name.
+This can be passed either as a string or as a `name: 'value'` key-value entry in a hash.
+
+See also the [AWS documentation on SSM Documents](https://docs.aws.amazon.com/systems-manager/latest/userguide/sysman-ssm-docs.html).
+
+
+## Properties
+
+|Property                     | Description|
+| ---                         | --- |
+|sha\_1                       | Provides the SHA1 hash of the document, which you can use for verification. |
+|hash                         | Provides the Sha256 or Sha1 hash created by the system when the document was created. |
+|hash\_type                   | Provides the hash type of the document. Valid values include Sha256 or Sha1. |
+|name                         | Provides the name of the Systems Manager document. |
+|version\_name                | Provides the version of the artifact associated with the document. |
+|owner                        | Provides the AWS user account that created the document. |
+|created\_date                | Provides the date when the document was created. |
+|status                       | Provides the status of the Systems Manager document. |
+|status\_information          | Provides a message returned by AWS Systems Manager that explains the Status value. |
+|document\_version            | Provides the document version. |
+|description                  | Provides a description of the document. |
+|parameters                   | Provides a description of the parameters for a document. |
```

Comment:
Could it be useful to mention that this a list of parameter items that contain `name`, `type`, `description`, `default_value`? https://docs.aws.amazon.com/systems-manager/latest/APIReference/API_DocumentParameter.html